docs/kit/logistics/logistics-software.md

```diff
@@ -0,0 +1,15 @@
+---
+original:
+  authors: Richard Barlow
+  url: https://srobo-legacy.gitbooks.io/student-robotics-kit-logistics/logistics-software.html
+---
+# Logistics Software
+
+To fulfil the multitude of responsibilities, the Kit Logistics Coordinator uses various pieces of custom software. The software projects used and maintained by the Kit Logistics Coordinator are listed below.
+
+## Software Projects
+
+| Name | Project URL | Notes |
```

Comment:
Formatting here is somewhat nasty.

Reply:
782420c improves this a bit, but I think the main limitation is markdown. I'm not aware of a syntax which allows wrapping within a column, which is what's really needed to make this good.